app/src/main/java/openfoodfacts/github/scrachx/openfood/fragments/SummaryProductFragment.java

```diff
@@ -338,7 +338,12 @@ public void onClick(View textView) {
         }
         if (isNotBlank(product.getCountries())) {
             countryProduct.setText(bold(getString(R.string.txtCountries)));
-            countryProduct.append(' ' + product.getCountries());
+            countryProduct.setClickable(true);
+            countryProduct.setMovementMethod(LinkMovementMethod.getInstance());
+            String[] countries = product.getCountries().split(",");
```

Comment:
Can you try splitting this on ", " rather than just ","? I see a lot of white spaces before the countries. I know that you may not have written this code

Reply:
@Karljoones I have only written that code. I will make the change .